packages/markdown/src/lib/MarkdownPlugin.ts

```diff
@@ -65,6 +66,12 @@ export type MarkdownConfig = PluginConfig<
      * @default null
      */
     allowNode?: AllowNodeConfig;
+    /**
+     * Custom options passed to remark-stringify.
+     *
+     * @default undefined
+     */
+    remarkStringifyOptions?: RemarkStringifyOptions;
```

Comment:
You'll need to set remarkStringifyOptions: null after L91, so the option is added to the plugin store.